docs/GettingStarted.md

```diff
@@ -12,8 +12,40 @@ your system, so that you can build apps with it right away. If you already
 have React Native installed, you can skip ahead to the
 [Tutorial](docs/tutorial.html).
 
-The instructions are a bit different depending on your development operating system, and whether you want to start developing for iOS or Android. If you
-want to develop for both iOS and Android, that's fine - you just have to pick
+## Quick Start
+
+[Create React Native App](https://github.com/react-community/create-react-native-app) is the easiest way to start building a new React Native application. It allows you to start a project without installing or configuring any native build tools.
+
+Assuming that you have Node installed, no Xcode or Android Studio installation is required:
+
+```
+npm install -g create-react-native-app
+create-react-native-app hello-world
+cd hello-world
+npm start
+```
+
+This will start a development server for you, and print a QR code in your terminal.
+
+Install the [Expo](https://expo.io) client app on your iOS or Android phone, make sure your phone is on the same network as your computer, and scan the QR code in your terminal to open your project. The QR scanner is available under the "Projects" tab of the app. Create React Native App also has a [user guide](https://github.com/react-community/create-react-native-app/blob/master/react-native-scripts/template/README.md) you can reference if you have questions specific to the tool.
+
+Once you've created your project and opened it in the Expo client app, you can proceed to the [Tutorial](docs/tutorial.html).
+
+### Caveats
```

Comment:
I think having this as a subsection makes sense because we want to surface the information for advanced users (read: the ones who are willing to read more than 3 sentences due to existing investment) early on without muddying the waters for really new people.

Very open to suggestions on the structure of this though.